### PR TITLE
add test cases

### PR DESCRIPTION
**Pull request by zhuangya**

### CODE REVIEW

File: src/utils/unflattenActionCreators.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ export default function unflattenActionCreators(
 ) {
   function unflatten(
     flatActionType,
-    partialNestedActionCreators = {},
-    partialFlatActionTypePath = []
+    partialNestedActionCreators,
+    partialFlatActionTypePath
```
[the remark this answers is not in the document]
Comment (zhuangya) [reply src/utils/unflattenActionCreators.js]: if with default values, test coverage would report these two lines as "uncoverd lines"

File: package.json [hunk 2]
```diff
@@ -68,6 +68,7 @@
     "invariant": "^2.2.1",
     "lodash.camelcase": "^4.3.0",
     "lodash.curry": "^4.1.1",
+    "lodash.ismap": "^4.4.2",
```
[the remark this answers is not in the document]
Comment (zhuangya) [reply package.json]: now i've rebased to the latest `origin/master` ;)

File: src/utils/unflattenActionCreators.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (philihp) [reply src/utils/unflattenActionCreators.js]: 👍

**Comment by timche** [comment]
Thanks a lot!

**Pull request merged**